File: rest/available-phone-numbers/local-basic-example-3/local-get-basic-example-3.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import TwilioRestClient

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "ACde6f132a3c49700934481addd5ce1659"
auth_token  = "{{ auth_token }}"
client = TwilioRestClient(account_sid, auth_token)

numbers = client.phone_numbers.search(country="US",
    contains="STORM",
    type="local")
# Purchase the first number in the list
if numbers:
    numbers[0].purchase()

File: rest/available-phone-numbers/local-basic-example-3/local-get-basic-example-3.js
// Download the Node helper library from twilio.com/docs/node/install
// These vars are your accountSid and authToken from twilio.com/user/account
var accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
var authToken = "your_auth_token";
var client = require('twilio')(accountSid, authToken);

client.availablePhoneNumbers("US").local.list({ 
    contains: "STORM" 
}, function(err, data) {
    for (var number in data.availablePhoneNumbers) {
        client.incomingPhoneNumbers.create({
            phoneNumber: number.phone_number
        }, function(err, purchasedNumber) {
            console.log(purchasedNumber.sid);
        });
    }
});